### ISSUE TITLE
IllegalStateException: @NotNull method de/endrullis/idea/postfixtemplates/language/CptUtil.getAbsoluteVirtualFile must not return null

### ISSUE THREAD

**Issue by sunrishe** [xylo/intellij-postfix-templates#78]
```
@NotNull method de/endrullis/idea/postfixtemplates/language/CptUtil.getAbsoluteVirtualFile must not return null
java.lang.IllegalStateException: @NotNull method de/endrullis/idea/postfixtemplates/language/CptUtil.getAbsoluteVirtualFile must not return null
	at de.endrullis.idea.postfixtemplates.language.CptUtil.$$$reportNull$$$0(CptUtil.java)
	at de.endrullis.idea.postfixtemplates.language.CptUtil.getAbsoluteVirtualFile(CptUtil.java:320)
	at de.endrullis.idea.postfixtemplates.language.CptUtil.getPath(CptUtil.java:306)
	at de.endrullis.idea.postfixtemplates.language.CptUtil.getLangAndVFile(CptUtil.java:371)
	at de.endrullis.idea.postfixtemplates.language.CptUtil.isUserTemplateFile(CptUtil.java:377)
	at de.endrullis.idea.postfixtemplates.intention.OverrideTemplateRuleIntention.isAvailable(OverrideTemplateRuleIntention.java:44)
	at com.intellij.codeInsight.intention.impl.ShowIntentionActionsHandler.availableFor(ShowIntentionActionsHandler.java:130)
	at com.intellij.codeInsight.daemon.impl.ShowIntentionsPass.lambda$getActionsToShow$2(ShowIntentionsPass.java:340)
	at com.intellij.codeInsight.intention.impl.ShowIntentionActionsHandler.chooseBetweenHostAndInjected(ShowIntentionActionsHandler.java:151)
	at com.intellij.codeInsight.daemon.impl.ShowIntentionsPass.getActionsToShow(ShowIntentionsPass.java:339)
	at com.intellij.codeInsight.daemon.impl.ShowIntentionsPass.getIntentionActionsToShow(ShowIntentionsPass.java:269)
	at com.intellij.codeInsight.daemon.impl.ShowIntentionsPass.doCollectInformation(ShowIntentionsPass.java:245)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:70)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$null$1(PassExecutorService.java:437)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1134)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:430)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:580)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:525)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:85)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:429)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:405)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:143)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:218)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:403)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:170)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
```

**Comment by xylo** [comment]
Fixed in version 2.0.4.

**Comment by sunrishe** [comment]
Surprise! Very fast processing
